src/main/java/org/tron/core/services/RpcApiService.java

```diff
@@ -381,6 +385,49 @@ public void getTransactionSign(TransactionSign req,
       responseObserver.onCompleted();
     }
 
+    @Override
+    public void createAdresss(BytesMessage req,
+        StreamObserver<BytesMessage> responseObserver) {
+      byte[] address = wallet.createAdresss(req.getValue().toByteArray());
+      BytesMessage.Builder builder = BytesMessage.newBuilder();
+      builder.setValue(ByteString.copyFrom(address));
+      responseObserver.onNext(builder.build());
+      responseObserver.onCompleted();
+    }
+
+    @Override
+    public void easyTransfer(EasyTransferMessage req,
+        StreamObserver<EasyTransferResponse> responseObserver) {
+      byte[] privateKey = wallet.pass2Key(req.getPassPhrase().toByteArray());
+      ECKey ecKey = ECKey.fromPrivate(privateKey);
+      byte[] owner = ecKey.getAddress();
+      TransferContract.Builder builder = TransferContract.newBuilder();
+      builder.setOwnerAddress(ByteString.copyFrom(owner));
+      builder.setToAddress(req.getToAddress());
+      builder.setAmount(req.getAmount());
+
+      TransactionCapsule transactionCapsule = null;
+      GrpcAPI.Return.Builder returnBuilder = GrpcAPI.Return.newBuilder();
+      EasyTransferResponse.Builder responseBuild = EasyTransferResponse.newBuilder();
+      try {
+        transactionCapsule = createTransactionCapsule(builder.build(),
+            ContractType.TransferContract);
+      } catch (ContractValidateException e) {
+        returnBuilder.setResult(false).setCode(response_code.CONTRACT_VALIDATE_ERROR)
+            .setMessage(ByteString.copyFromUtf8(e.getMessage()));
+        responseBuild.setResult(returnBuilder.build());
+        responseObserver.onNext(responseBuild.build());
+        responseObserver.onCompleted();
+      }
+
+      transactionCapsule.sign(privateKey);

```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "transactionCapsule" is nullable here. ([squid:S2259](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2259))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1050&issues=AWQcYldcbXXTsx7O_qQD&open=AWQcYldcbXXTsx7O_qQD)

[ISSUE_KEY]: AWQcYldcbXXTsx7O_qQD